src/Bicep.Core/Semantics/Namespaces/AzNamespaceType.cs

```diff
@@ -206,49 +206,59 @@ private static IEnumerable<(FunctionOverload functionOverload, ResourceScope all
             yield return (
                 new FunctionOverloadBuilder("tenant")
                     .WithDynamicReturnType(GetTenantReturnType, new TenantScopeType(Enumerable.Empty<FunctionArgumentSyntax>(), Enumerable.Empty<TypeProperty>()))
+                    .WithGenericDescription("Returns the current tenant scope.")
```

Comment:
>Returns the current tenant scope. [](http://example.com/codeflow?start=45&length=33)

Docs have this as "Returns properties about the tenant for the current deployment." which seems like a better description: [https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-scope#tenant](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-scope#tenant)